scripts/mods/msu/popup.nut

```diff
@@ -1,45 +1,58 @@
 this.popup <- {
 	m = {
 		JSHandle = null,
-		TextCache = "",
-		ForceQuit = false
+		TextCache = [],
 	}
 
-	function showRawText( _text, _forceQuit = false )
+	function addLine(_text)
 	{
-		if (_forceQuit) this.m.ForceQuit = true;
-		if (this.m.JSHandle == null)
-		{
-			if (this.m.TextCache != "") this.m.TextCache += "<br>";
-			this.m.TextCache += _text;
-		}
-		else
+		this.m.JSHandle.asyncCall("addLine", _text);
+	}
```

Comment:
The caching component has been completely removed here which is required for the popup to work